fmriprep/cli/run.py

```diff
@@ -219,7 +219,6 @@ def main():
             config.execution.fmriprep_dir,
             config.execution.run_uuid,
             config=data.load("reports-spec.yml"),
-            packagename="fmriprep",
         )
```

Comment:
```suggestion
        )
        if failed_reports:
            config.loggers.cli.error(
                "Report generation was not successful for the following participants : %s.",
                ", ".join(failed_reports),
            )
```

Reply:
@oesteban l.226-231 below already log the error if failed reports, but using sentry_sdk, which logger should we keep sentry_sdk or config.loggers?

Reply:
sentry is doing a different thing: it sends telemetry. So we need both.